Python/initconfig.c

```diff
@@ -180,139 +180,134 @@ arg ...: arguments passed to program in sys.argv[1:]\n\
-"   os.cpu_count(), and multiprocessing.cpu_count() if set to a positive integer.\n"
-"PYTHONDEVMODE: enable the development mode.\n"
+"                  os.cpu_count(), and multiprocessing.cpu_count() if set to\n"
+"                  a positive integer.\n"
+"PYTHONDEVMODE   : enable the development mode.\n"
 "PYTHONPYCACHEPREFIX: root directory for bytecode cache (pyc) files.\n"
 "PYTHONWARNDEFAULTENCODING: enable opt-in EncodingWarning for 'encoding=None'.\n"
-"PYTHONNODEBUGRANGES: if this variable is set, it disables the inclusion of the \n"
-"   tables mapping extra location information (end line, start column offset \n"
-"   and end column offset) to every instruction in code objects. This is useful \n"
-"   when smaller code objects and pyc files are desired as well as suppressing the \n"
-"   extra visual location indicators when the interpreter displays tracebacks.\n"
-"PYTHON_FROZEN_MODULES   : if this variable is set, it determines whether or not \n"
-"   frozen modules should be used. The default is \"on\" (or \"off\" if you are \n"
-"   running a local build).\n"
-"PYTHON_COLORS           : If this variable is set to 1, the interpreter will"
-"   colorize various kinds of output. Setting it to 0 deactivates this behavior.\n"
-"PYTHON_HISTORY          : the location of a .python_history file.\n"
-"These variables have equivalent command-line parameters (see --help for details):\n"
-"PYTHONDEBUG             : enable parser debug mode (-d)\n"
-"PYTHONDONTWRITEBYTECODE : don't write .pyc files (-B)\n"
-"PYTHONINSPECT           : inspect interactively after running script (-i)\n"
-"PYTHONINTMAXSTRDIGITS   : limit max digit characters in an int value\n"
-"                          (-X int_max_str_digits=number)\n"
-"PYTHONNOUSERSITE        : disable user site directory (-s)\n"
-"PYTHONOPTIMIZE          : enable level 1 optimizations (-O)\n"
-"PYTHONSAFEPATH          : don't prepend a potentially unsafe path to sys.path (-P)\n"
-"PYTHONUNBUFFERED        : disable stdout/stderr buffering (-u)\n"
-"PYTHONVERBOSE           : trace import statements (-v)\n"
-"PYTHONWARNINGS=arg      : warning control (-W arg)\n"
+"PYTHONNODEBUGRANGES: if this variable is set, it disables the inclusion of\n"
+"                  the tables mapping extra location information (end line,\n"
+"                  start column offset and end column offset) to every\n"
+"                  instruction in code objects.  This is useful when smaller\n"
+"                  code objects and pyc files are desired as well as\n"
+"                  suppressing the extra visual location indicators when the\n"
+"                  interpreter displays tracebacks.\n"
+"PYTHON_FROZEN_MODULES: if this variable is set, it determines whether or not\n"
+"                  frozen modules should be used.  The default is \"on\" (or\n"
+"                  \"off\" if you are running a local build).\n"
+"PYTHON_COLORS   : If this variable is set to 1, the interpreter will colorize\n"
```

Comment:
```suggestion
"PYTHON_COLORS   : if this variable is set to 1, the interpreter will colorize\n"
```

(Yes, there are period and upper-case in the next line; that’s consistent with lines 283, 288, etc.)